cpp/src/comms/ipc/ipc.cu

```diff
@@ -16,6 +16,9 @@
 #include <vector>
 #include <string>
 
+//TODO fixed 0.11.1 ... delete all uses of older arrow API (< 800)
+#define ARROW_VERSION 11*1000 + 1*100
```

Comment:
This should not be hardcoded in the source code. It should be configured in the CMake files.

Reply:
Allright!